### PR TITLE
bump everything to current versions

### PR DESCRIPTION
**Pull request by zWhdmB5T**

### CODE REVIEW

File: org.gnome.glabels-3.desktop
Comment (hfiguiere) [inline org.gnome.glabels-3.desktop:1]: why was this added?

File: org.gnome.glabels-3.metainfo.xml
Comment (hfiguiere) [inline org.gnome.glabels-3.metainfo.xml:1]: why was this added?

File: org.gnome.glabels-3.json [hunk 1]
```diff
@@ -155,18 +155,9 @@
...
-                },
-                {
-                    "type": "patch",
-                    "path": "appdata.patch"
-                },
-                {
-                    "type": "patch",
-                    "path": "glabels-fixes.patch"
+                    "type": "archive",
+                    "url": "https://github.com/jimevins/glabels-qt/archive/refs/tags/glabels-3.99-master564.tar.gz",
```
Comment (hfiguiere) [inline org.gnome.glabels-3.json:159]: this is not a stable release.

Flathub is NOT for nightly release. Even less for switching like that.

File: org.gnome.glabels-3.desktop
Comment (zWhdmB5T) [reply org.gnome.glabels-3.desktop:1]: standard use — how else?

File: org.gnome.glabels-3.metainfo.xml
Comment (zWhdmB5T) [reply org.gnome.glabels-3.metainfo.xml:1]: standard use — how else?

File: org.gnome.glabels-3.json (hunk 1 above)
Comment (zWhdmB5T) [reply org.gnome.glabels-3.json:159]: this release is the v4 RC…
no changes since them, seems not maintained any more
previous version 3.4.1 is really quite old…

File: org.gnome.glabels-3.desktop
Comment (hfiguiere) [reply org.gnome.glabels-3.desktop:1]: no it's not standard to add them.

File: org.gnome.glabels-3.metainfo.xml
Comment (hfiguiere) [reply org.gnome.glabels-3.metainfo.xml:1]: no it's not standard to add them. They are already upstream.

**Comment by zWhdmB5T** [comment]
I can cancel it if you want — I just wanted the most recent version. Currently I run exactly this version as AppImage and it is stable, indeed

File: org.gnome.glabels-3.json (hunk 1 above)
Comment (hfiguiere) [reply org.gnome.glabels-3.json:159]: it is a different repository. So this should be submitted as a NEW app. Even according the the name it uses a different toolkit.

Comment (zWhdmB5T) [reply org.gnome.glabels-3.json:159]: Would you like to do the job? I would appreciate it! I just did this because I would like to switch from AppImage to FlatPak. But this specific FlatPak is quite outdated. The version of gLabels is 3.4.1 and is outdated by far. The current version is 3.99+master364, it is version 4 RC and it is stable as AppImage. The included libraries in this v3.4.1 FlatPak are also very old, i.e. outdated.
